blackspots/blackspots.json

```diff
@@ -0,0 +1,68 @@
+{
+  "type": "dataset",
+  "id": "blackspots",
+  "title": "blackspots",
+  "status": "beschikbaar",
+  "description": "Blackspots",
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "blackspots",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "schema",
+          "id",
+          "locatie id",
+          "spot type",
+          "geometry",
+          "stadsdeel",
+          "status"
+        ],
+        "display": "locatie id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "integer"
+          },
+          "locatie id": {
+            "type": "string"
+          },
+          "spot type": {
+            "type": "string"
```

Comment:
Is dit niet eigenlijk een enum?

Reply:
Goed punt, ga ik aanpassen. Het gaat om:
-  protocol ernstig
- blackspot
 - risico
 - protocol dodelijk
 - wegvak

Overigens kent Django qua modelgeneratie geen enums. Dus op postgres niveau wordt dit gewoon een charfield.